File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q32.js
import { question } from "readline-sync";
function main(){
    //Input
    const num = question("Enter a 3 digit number: ")

    //Processing

    const first_number = Math.floor(num / 100)
    const second_number = Math.floor((num % 100) / 10)
    const third_number = (num % 100) % 10

    const inverse_number = third_number * 100 + second_number * 10 + first_number

    const subtraction_result = num - inverse_number

    //Output
    console.log(`The result of the subtraction of the entered number (${num}) by its inverse (${inverse_number}) is equal to: ${subtraction_result}`)

}
main()

File: ADS 2023.2/AlgoritimosProgramacao/Algoritmos/Questões/List-01/q32.py
def main():
    #Input
    num = int(input("Enter a 3-digit integer value: "))

    #Processing

    first_number = round(num / 100)
    second_number = round((num % 100) / 10)
    third_number = (num % 100) % 10

    inverse_number = (third_number * 10 ** 2 + second_number * 10 ** 1 + first_number * 10 ** 0)

    result = num - inverse_number

    #Output
    print(f"The result of the sum of the entered number ({num}) and its inverse ({inverse_number}) is equal to: {result}")

main()